### DISCUSSION TITLE
Ability to pick directory in picker/explorer mode

### DISCUSSION THREAD

**Discussion by jpalus** [jarun/nnn#1919] [Q&A]
If I understand correctly it is is not currently possible to get directory in neither picker (`nnn -p`) nor explorer (`nnn -F1`) mode. Would you consider adding such support? Possibly a binding that publishes file path under cursor irrespective of its type to file and closes `nnn` (picker) or writes to fifo and continues running (explorer).

**Comment by darukutsu** [comment]
select directories you want to share, then press q

**Reply by jpalus** [reply]
For the time being  I'll take above answer as "NO" but it's definitely not an answer:
> writes to fifo and continues running (explorer).

after `q` it definitely does not continue running.

As for picker mode it does meet criteria either. In 99% of cases I don't want to select anything but just use what is hovered. I admit this for `nnn` use as a `vim` plugin. Requirement to select instead of just hovering ruins experience.

**Reply by jarun** [reply]
> In 99% of cases I don't want to select anything but just use what is hovered

Select file/dir and press `Q` (capital).
You _have_ to select, just hovering on it won't work. That's because another user may just want to quit.
